gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-default-page/api-documentation-v4-default-page.component.ts

```diff
@@ -157,12 +157,19 @@ export class ApiDocumentationV4DefaultPageComponent implements OnInit, OnDestroy
       )
       .subscribe({
         next: (_) => {
-          this.snackBarService.success('Page deleted successfully');
+          this.snackBarService.success('Homepage deleted successfully');
           this.ngOnInit();
         },
         error: (error) => {
-          this.snackBarService.error(error?.error?.message ?? 'Error while deleting page');
+          this.snackBarService.error(error?.error?.message ?? 'Error while removing homepage');
         },
       });
   }
+
+  chooseHomepage() {
+    this.router.navigate(['.', 'homepage', 'choose'], {
+      relativeTo: this.activatedRoute,
+      queryParams: { parentId: this.parentId },
```

Comment:
Why do you need the parentId for choosing the homepage?

Reply:
Done